aws/resource_aws_sagemaker_model_test.go

```diff
@@ -113,21 +97,24 @@ func TestAccAWSSagemakerModel_tags(t *testing.T) {
 		CheckDestroy: testAccCheckSagemakerModelDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccSagemakerModelConfigTags(rName),
+				Config: testAccSagemakerModelConfigTags(rName, image),
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckSagemakerModelExists("aws_sagemaker_model.foo"),
```

Comment:
Hey @jckuester, was there any reason why this and the corresponding method was removed? It's a fairly important check for the tests

Reply:
Sorry, then is misunderstood. But thinking of it again now, this check makes totally sense: 1) We need to first check if the resource is properly created in AWS and after that we check whether the terraform state is correct. Btw, I started working on the reviews for the other resources, too, but haven't had much time recently.